app/mailers/jobseekers/devise_emails.rb

```diff
@@ -15,7 +15,7 @@ def confirmation_instructions(record, token, _opts = {})
     send_email(
       jobseeker: record,
       subject: subject,
-      template: NOTIFY_JOBSEEKER_CONFIRMATION_TEMPLATE,
```

Comment:
seeing as these 4 methods are almost identical, could they not be refactored into 1 method with the token being optional arg where its needed?

Reply:
I thought about when changing it. 
I ended up not extracting the method, because I didn't think it'd be that much of a benefit as `send_email` is already that method.

I'm going to have to do a last PR cleanup after I deploy this and I'll try extracting it and see if it's better 🤓